console/src/components/UIKit/atoms/AlertBox/index.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { theme } from '../../theme';
+import { Icon } from '../../Icons';
+
+import { AlertBoxStyles } from './Alert.style';
+import { Text } from '../Typography';
+
+// ***************************** //
+
+const AlertBox = props => {
+  const { children, type } = props;
+
+  /* Color values dynamically assigned from theme object based on alert type.
+   *  - backgroundColor
+   *  - borderColor
+   *  If the alert type is out of the range then default color value is assigned.
```

Comment:
Can you remove this comment? Nothing is out of range, it's just default. It's super common -- you have some default component and then some variations based on the passed props. It doesn't mean something is out of range, it's just a default component.

Reply:
Yes, I'll remove.